qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -111,7 +112,7 @@ def study_get_req(study_id, user_id):
     # Study.has_access no_public=True, will return True only if the user_id is
     # the owner of the study or if the study is shared with the user_id
     study_info['has_access_to_raw_data'] = study.has_access(
```

Comment:
The above comment is no longer true, right? Now, if the data is considered public, it will also return True.